.github/CODEOWNERS

```diff
@@ -148,6 +148,9 @@
 
 
 # Management Plane
+# PRLabel: %Mgmt
+/sdk/**/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
```

Comment:
Unfortunately wildcard syntax will not work for the scenarios we have. There are some limitations in some of the tools when resolving these. For example our pipeline notification tool does not currently support this wildcard syntax.

Reply:
Now supported:
- https://github.com/Azure/azure-sdk-tools/issues/2770#issuecomment-1433869882